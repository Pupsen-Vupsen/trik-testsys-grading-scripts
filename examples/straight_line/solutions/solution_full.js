var __interpretation_started_timestamp__;
var pi = 3.141592653589793;

var main = function()
{
	__interpretation_started_timestamp__ = Date.now();
	
	brick.motor(M3).setPower(100);
	brick.motor(M4).setPower(100);
	
	script.wait(1250);
	
	while (true) {
		if (brick.sensor(A1).read() < 10) {
			break;
		}
		if (brick.sensor(A1).read() < 40) {
			script.wait(500);
			
			brick.motor(M1).brake();
			brick.motor(M2).brake();
			brick.motor(M3).brake();
			brick.motor(M4).brake();
			
			script.wait(1000);
			
			brick.motor(M3).setPower(100);
			brick.motor(M4).setPower(100);
			
			script.wait(500);
			
		}
	}
	script.wait(500);
	
	brick.motor(M1).brake();
	brick.motor(M2).brake();
	brick.motor(M3).brake();
	brick.motor(M4).brake();
	
	return;
}
