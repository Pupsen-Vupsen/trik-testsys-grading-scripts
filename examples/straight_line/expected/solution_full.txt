-------- Results for solution_full --------
1_square.json: 100
2_square.json: 100
TOTAL: 200
